api/src/test/java/org/pmiops/workbench/elasticsearch/ElasticFiltersTest.java

```diff
@@ -782,15 +808,17 @@ public void testAgeQuery() {
   }
 
   @Test
-  public void testAgeAndVisitQuery() {
+  public void testAgeAndEthnicityQuery() {
     OffsetDateTime now = OffsetDateTime.now(ZoneOffset.UTC);
     Object left = now.minusYears(34).minusYears(1).toLocalDate();
     Object right = now.minusYears(20).toLocalDate();
     SearchParameter ageParam =
         new SearchParameter()
-            .type(TreeType.DEMO.toString())
-            .subtype(TreeSubType.AGE.toString())
+            .domain(DomainType.PERSON.toString())
+            .type(CriteriaType.AGE.toString())
             .group(false)
+            .ancestorData(false)
```

Comment:
I don't see this value being read in ElasticFilters; should it be? Or is that a different ticket?

Reply:
ancestor data property is referenced several times by the CriteriaLookupUtil which ElasticFilters is now using. 
https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/cohortbuilder/querybuilder/util/CriteriaLookupUtil.java#L107